src/Infrastructure/Client/Config.ts

```diff
@@ -1,10 +1,9 @@
 import * as fs from 'fs';
-import * as yargs from 'yargs';
 import * as JSON5 from 'json5';
 import Configuration from '../System/Configuration';
 
 /** @internal */
-const rawConfig: string = fs.readFileSync(yargs.argv.config as string, 'utf8');
+const rawConfig: string = fs.readFileSync('./config.json5' as string, 'utf8');
```

Comment:
Did we decide to hardcode this so we could get rid of yargs?

Reply:
I was getting errors with the yargs build, also it seemed like hard-coding it wouldn't be an issue.  One less dep.